scss/_variables.scss

```diff
@@ -902,8 +902,8 @@ $form-validation-states: (
 $zindex-dropdown:                   1000 !default;
 $zindex-sticky:                     1020 !default;
 $zindex-fixed:                      1030 !default;
-$zindex-offcanvas:                  1040 !default;
-$zindex-modal-backdrop:             1050 !default;
+$zindex-modal-backdrop:             1040 !default;
```

Comment:
Does it make sense to rename this into backdrop since its standalone now?

Reply:
I would, as it would made sense to keep it in separate of modal.scss too, but in previous Pr I was advised to keep it as it is, till stable launch. So I just want to 'borrow' the functionality